css/main.css

```diff
@@ -27,7 +27,35 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+#desktop-nav {
+  display: none;
+}
+
+#desktop-nav.nav-open {
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background: #3c3a39;
+  z-index: 1;
+}
```

Comment:
- [ ]  Please give to the `desktop-nav`  a  `position: fixed; property ` instead of `position: absolute;'  to prevent scrolling the navigation mobile menu every time the user clicks (or taps) the hamburger button because currently, users can scroll the screen and leave the mobile menu fixed to the up which should not be the case 😢